src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_controlnet.py

```diff
@@ -918,6 +918,13 @@ def __call__(
         if isinstance(self.controlnet, MultiControlNetModel) and isinstance(controlnet_conditioning_scale, float):
             controlnet_conditioning_scale = [controlnet_conditioning_scale] * len(self.controlnet.nets)
 
+        global_pool_conditions = (
+            self.controlnet.config.global_pool_conditions
+            if isinstance(self.controlnet, ControlNetModel)
+            else self.controlnet.nets[0].config.global_pool_conditions
```

Comment:
This breaks torch compile since `self.controlnet` becomes an instance of `torch._dynamo.eval_frame.OptimizedModule`. I tried to run the benchmark script from https://github.com/huggingface/diffusers/pull/3313

Reply:
Looking! Thanks for the flag

Reply:
Handled it here: https://github.com/huggingface/diffusers/pull/3386 which should be merged today